.devcontainer/on-create-command.sh

```
@@ -3,12 +3,12 @@ set -ex
 
 # For some reason the directory is not setup correctly and causes build of devcontainer to fail since
 # it doesn't have access to the workspace directory. This can normally be done in post-start-command
-git config --global --add safe.directory /workspaces/ephemeral-pulumi-deploy
+script_dir="$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)